spec/json/tsg_email_validation_v3.json

```
@@ -225,10 +225,427 @@
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "operationId": "GetValidationsEmailJobs",
+        "summary": "This request lists all of a User's Bulk Email Validation Jobs.",
+        "tags": [
+          "Bulk Email Address Validation"
+        ],
+        "description": "**This endpoint returns a list of all of your Bulk Email Validation Jobs.**\n",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GetValidationsEmailJobs200Response"
+                },
+                "examples": {
+                  "response": {
+                    "value": {
+                      "result": [
+                        {
+                          "id": "01HV9ZZQAFEXW18KFEPTB9YD5E",
+                          "status": "Queued",
+                          "started_at": 1712954639,
+                          "finished_at": 0
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorResponse"
+                },
+                "examples": {
+                  "response": {
+                    "value": {
+                      "errors": [
+                        {
+                          "message": "error message"
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/v3/validations/email/jobs/{job_id}": {
+      "parameters": [
+        {
+          "name": "job_id",
+          "in": "path",
+          "description": "The ID of the Bulk Email Address Validation Job you wish to retrieve.",
+          "required": true,
+          "schema": {
+            "type": "string"
+          }
+        }
+      ],
+      "get": {
+        "operationId": "GetValidationsEmailJobsJobId",
+        "summary": "This request returns a single Bulk Email Validation Job.",
+        "tags": [
+          "Bulk Email Address Validation"
+        ],
+        "description": "**This endpoint returns a specific Bulk Email Validation Job. You can use this endpoint to check on the progress of a Job.**\n",
+        "responses": {
+          "200": {
+            "description": "",
```

```suggestion
            "description": "The request was successful. The response contains a specific Bulk Email Validation Job.",
```
If detailed description is helpful